src/modules/Login/action.js

```diff
@@ -193,8 +193,9 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
 export const mergeSettings = (
   loadedSettings: Object,
   defaults: Object,
-  types: Object
-): { finalSettings: Object, isOverwriteNeeded: boolean, isDefaultTypeIncorrect: boolean } => {
+  types: Object,
+  account?: Object
+): { finalSettings: Object, isOverwriteNeeded: boolean, isDefaultTypeIncorrect: boolean, account?: Object } => {
```

Comment:
It doesn't look like you ever return account but it's in the flow type

Reply:
Ah thank you for the heads up, I was getting the param types mixed up with the return types.